src/Auditable.php

```diff
@@ -313,7 +318,33 @@ protected function isAttributeAuditable($attribute)
      */
     protected function isEventAuditable($event)
     {
-        return in_array($event, $this->getAuditableEvents());
+        return $this->getEventAttributesMethodName($event) !== null;
+    }
+
+    /**
+     * Get Event Attributes Method Name.
+     *
+     * @param string $event
+     *
+     * @return mixed The event's custom attribute method name if it has one, false it has none or null if the event is
+     *               not auditable.
+     */
+    private function getEventAttributesMethodName($event)
+    {
+        foreach ($this->getAuditableEvents() as $key => $value) {
+            $auditableEventName = is_int($key) ?
+                $value :
+                $key;
+
+            // replace wildcards with .* which is regex for "any character"
```

Comment:
Remove comment, this one is a bit obvious :)

Reply:
haha sure, you never know... I'll remove it!